### PR TITLE
docs: add owner_type queryparam to swagger docs

### PR DESCRIPTION
**Pull request by mariajgrimaldi**
This PR allows us to filter using owner_type in swagger api-docs

### CODE REVIEW

File: eox_tagging/api/v1/viewset.py [hunk 1]
```diff
@@ -123,10 +123,16 @@
...
+        query_parameter(
+            "owner_type",
+            str,
+            "Shortcut to filter objects of owner_type `user` or `site`.",
+        ),
         query_parameter(
             "target_type",
             str,
-            "The type of the object that was tagged, one of: `course`, `courseenrollment`, `user`",
+            "The type of the object that was tagged, one of: `course (opaquekeyproxymodel)`"
```
Comment (MoisesGSalas) [inline eox_tagging/api/v1/viewset.py:134]: How many people actually know what `opaquekeyproxymodel` means?

Comment (mariajgrimaldi) [reply eox_tagging/api/v1/viewset.py:134]: Not many. My issue here is: if you filter using just `course` it's not going to work.

Comment (mariajgrimaldi) [reply eox_tagging/api/v1/viewset.py:134]: Let me rephrase the change

**Comment by mariajgrimaldi** [comment]
We have to solve filtering by opaque key proxy model, but not in this PR. @MoisesGSalas

**Review by MoisesGSalas — APPROVED** [review]
